changes/130.fixed

```diff
@@ -0,0 +1 @@
+Fixed `invoke tests` exiting early even when tests pass.
```

Comment:
This filename references PR 130 rather than an issue. If there isn't an issue, you can use this PR number (133) instead.

Also, this should probably be `.housekeeping` or `misc` as it doesn't really relate to the library that gets released, only dev tests.